### PR TITLE
Submodule Bulma repo instead of cloning

### LINKED ISSUE #42: Submodule bulma instead of keeping a copy.
Right now, we've got copies of specific versions of Bulma and Bulma-Extensions in this repo, which is license permissable but adds maintenance in that we have to manually update these files to in order to "update" to a new version.

Perhaps we could include bulma and bulma-extensions as submodules to ensure these stayed up to date, and that we did not need to manage these files ourselves?

### PR DESCRIPTION
**Pull request by JonasUJ**
Adds Bulma as a submodule, meaning manual updates wont be necessary any more. This almost closes #42. Now it is just missing the extensions.

### CODE REVIEW

**Comment by lemonsaurus** [comment]
Thanks for doing this!

I don't think I want to merge this into master until we've solved bulma extensions submoduling and PyPI builds, but maybe we can merge it into a feature branch and continue work there.

**Comment by lemonsaurus** [comment]
Created a new feature branch for this now. Can you target `submodule_refactor` instead of `master`

**Comment by JonasUJ** [comment]
Done :)

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,4 @@
 include LICENSE
 recursive-include django_simple_bulma *
+prune django_simple_bulma/bulma
+graft django_simple_bulma/bulma/sass
```
Comment (lemonsaurus) [inline MANIFEST.in:4]: Can you explain this code? Also, maybe there should be a newline at the end of this file.

File: django_simple_bulma/finders.py [hunk 2]
```diff
@@ -40,26 +41,37 @@ def __init__(self):
...
-        scss_string = ""
+        scss_string = '@charset "utf-8";\n'
         for var, value in self.variables.items():
             scss_string += f"${var}: {value};\n"
 
         # SASS wants paths with forward slash:
         sass_bulma_path = str(self.simple_bulma_path).replace('\\', '/')
-        # Now load bulma
-        scss_string += f'@import "{sass_bulma_path}/bulma.sass";'
+        sass_bulma_submodule_path = str(self.bulma_submodule_path).replace('\\', '/')
```
Comment (lemonsaurus) [inline django_simple_bulma/finders.py:50]: I think pathlibs `.as_posix()` method solves this, actually. Could you test if it works if we just do `self.bulma_submodule_path.as_posix()`?

If this works, do that for the other replaces too?

**Comment by JonasUJ** [comment]
If we want the extensions to be submodules too, we probable have to do it like these people have https://github.com/Wikiki/bulma-extensions. It shouldn't be too much trouble, since all the extensions (afaik) have this structure "/src/sass/" and that would just have to be isolated.

About the PyPi builds, running `python setup.py sdist bdist_wheel` only includes the "/bulma/sass/" directory in the "build/" because I added these two lines to the MANIFEST.in
```
prune django_simple_bulma/bulma
graft django_simple_bulma/bulma/sass
```

**Comment by lemonsaurus** [comment]
Excellent, that's very exciting. Then this may be closer to releasable than I thought.

File: django_simple_bulma/finders.py (hunk 2 above)
Comment (JonasUJ) [reply django_simple_bulma/finders.py]: There is one replace call further down that cannot be replaced because it is used on a str instead of a Path.

**Pull request merged**